scripts/run_frontend_tests.py

```diff
@@ -74,9 +75,19 @@
 )
 _PARSER.add_argument(
     '--download_combined_frontend_spec_file',
-    help='optional; if specifided, downloads the combined frontend file',
+    help='optional; if specified, downloads the combined frontend file',
     action='store_true'
 )
+_PARSER.add_argument(
+    '--run_on_changed_files',
+    help='optional; if specified, runs the frontend karma tests on the '
+    'files that were changed in the current branch.',
+    action='store_true'
+)
+_PARSER.add_argument(
+    '--specs_to_run',
+    help='optional; if specified, runs the specified test specs'
```

Comment:
Specify that this should be a comma-separated list of file paths

Reply:
Done.